docs/pages/enterprise/getting-started.mdx

```diff
@@ -259,7 +250,7 @@ $ sudo tctl create -f role.yaml
 Now, lets create a new Teleport user "joe" with "access" role:
 
 ```bsh
-$ sudo tctl users add --roles=access joe
+$ sudo tctl users add --roles=access joe --logins=joe,ubuntu,ec2-user
```

Comment:
Can you put `--logins` before `joe`. It's easy for the argument to "get lost" when it's placed in between flags like this.